internal/config/config.go

```diff
@@ -570,6 +576,11 @@ func bgpAdvertisementFromCR(crdAd metallbv1beta1.BGPAdvertisement, communities m
 
 	ad.LocalPref = crdAd.Spec.LocalPref
 
+	if len(crdAd.Spec.Peers) > 0 {
+		ad.Peers = make([]string, 0, len(crdAd.Spec.Peers))
+		ad.Peers = append(ad.Peers, crdAd.Spec.Peers...)
```

Comment:
Can you extend the unit tests with this?

Reply:
what do you mean?

Reply:
I mean, this part is not covered by unit tests (I think).